packages/snap/src/Snap.service.ts

```diff
@@ -545,11 +547,15 @@ class SnapService {
       case 'togglePopups':
         res = await GeneralService.togglePopups();
         return ResultObject.success(res);
-      case 'addFriendlyDapp':
-        await GeneralService.addFriendlyDapp({ id: this.origin });
+      case 'addTrustedDapp':
+        if (origin === 'masca.io')
```

Comment:
What happens on `dev.masca.io` and `beta.masca.io`

Reply:
I think it should work since the domain `masca.io` gets set and checked, but we'll see.